content/docs/deployment/csmoperator/drivers/powerstore.md

```diff
@@ -0,0 +1,94 @@
+   type: Opaque
+   data:
+      config: CONFIG_YAML
+   ```
+
+   Combine both files and create Kubernetes secret by running the following command:
+   ```bash
+   sed "s/CONFIG_YAML/`cat config.yaml | base64 -w0`/g" secret.yaml | kubectl apply -f -
+   ```
+
+### Install Driver
+
+1. Follow all the [prerequisites](#prerequisite) above
+   
+2. Create a CR (Custom Resource) for PowerStore using the sample files provided 
+   [here](https://github.com/dell/csm-operator/tree/master/samples). This file can be modified to use custom parameters if needed.
+
+3. Users should configure the parameters in CR. The following table lists the primary configurable parameters of the PowerStore driver and their default values:
+
+  | Parameter | Description | Required | Default |
+| --------- | ----------- | -------- |-------- |
+| replicas | Controls the number of controller pods you deploy. If the number of controller pods is greater than the number of available nodes, the excess pods will be pending state till new nodes are available for scheduling. Default is 2 which allows for Controller high availability. | Yes | 2 |
+| namespace | Specifies namespace where the drive will be installed | Yes | "test-powerstore" |
+| fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType` | No |"ReadWriteOnceWithFSType"|
+| storageCapacity | Enable/Disable storage capacity tracking feature | No | true |
+| ***Common parameters for node and controller*** |
+| X_CSI_POWERSTORE_NODE_NAME_PREFIX | Prefix to add to each node registered by the CSI driver | Yes | "csi-node" 
+| X_CSI_FC_PORTS_FILTER_FILE_PATH | To set path to the file which provides a list of WWPN which should be used by the driver for FC connection on this node | No | "/etc/fc-ports-filter" |
+| ***Controller parameters*** |
+| X_CSI_POWERSTORE_EXTERNAL_ACCESS | allows specifying additional entries for hostAccess of NFS volumes. Both single IP address and subnet are valid entries | No | " "|
+| X_CSI_NFS_ACLS | Defines permissions - POSIX mode bits or NFSv4 ACLs, to be set on NFS target mount directory. | No | "0777" |
+| ***Node parameters*** |
+| X_CSI_POWERSTORE_ENABLE_CHAP | Set to true if you want to enable iSCSI CHAP feature | No | false |
+
+4.  Execute the following command to create PowerStore custom resource:`kubectl create -f <input_sample_file.yaml>`. The above command will deploy the CSI PowerStore driver in the namespace specified in the input YAML file
+      - After that the driver should be installed, you can check the condition of driver pods by running `kubectl get all -n <driver-namespace>`
+
+5.  [Verify the CSI Driver installation](../#verifying-the-driver-installation)
+    
+**Note** : 
+   1. "Kubelet config dir path" is not yet configurable in case of Operator based driver installation.
+   2. Also, snapshotter and resizer sidecars are not optional to choose, it comes default with Driver installation. 
```

Comment:
Delete "Also,". Initial uppercase on Snapshotter. Delete "to choose," Add full stop ("." period) after "optional". Replace "it" with "They". Replace "it comes" with "are defaults".

Reply:
Done